.github/workflows/pre-release.yml

```diff
@@ -1,6 +1,9 @@
 name: Pre-release
 
-on: workflow_dispatch
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
We probably want to filter to run this only on change on the `packages/open-next` folder itself.
Really not a big deal but change made to e2e would trigger this which is not really useful

Reply:
good point! 👍